src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -76,6 +78,10 @@ public function configureOptions(OptionsResolver $resolver)
                 'window.delete' => $this->trans('Delete selection', 'Admin.Actions'),
                 'window.close' => $this->trans('Close window', 'Admin.Actions'),
                 'window.zoom' => $this->trans('Zoom on selection', 'Admin.Actions'),
+                'modal.close' => $this->trans('Cancel', 'Admin.Actions'),
+                'modal.accept' => $this->trans('Delete', 'Admin.Actions'),
+                'modal.title' => $this->transChoice('Are you sure you want to delete the selected image?|Are you sure you want to delete the %filesNb% selected images?', 'Admin.Catalog.Notification'),
```

Comment:
Oh by the way @NeOMakinG I suggested this type of translations In twig we can handle pluralization, and it seems like in VueI18n as well

https://symfony.com/doc/3.3/components/translation/usage.html#pluralization
https://kazupon.github.io/vue-i18n/guide/pluralization.html#pluralization

Both syntaxes are similar, except in twig the separator is `|` whereas in VueI18n it seems to be ` | ` (additional spaces), but the principal is the same.

The trick here is that we inject the wording into the js component so we can't use `transChoice` here in the php file (nor could we in a twig template) since we don't have the number of elements at this moment (which is dynamic in the JS component). So it's in the Vue component that we need to use `$tc('modal.title'), filesNb, { filesNb: filesNb }` (first `filesNb` is used to switch the wording from singular to plural, second one is to replace placeholders)

The initial wording here might need some changes, as in the example of Vue it uses `{count}` and not `%count%` I wonder if we could use `$tc('modal.title'), filesNb, { '%filesNb%': filesNb }` maybe?

Reply:
Any way this needs a few trials to make sure:
- which separator use as input, it could be replaced in the twig
- which format of placeholder we need to use (might be harder to change in twig so the easiest is tu use the right one in the wording I think)

Reply:
```suggestion
                'modal.title' => $this->trans('Are you sure you want to delete the selected image?|Are you sure you want to delete the %filesNb% selected images?', 'Admin.Catalog.Notification'),
```

Reply:
```suggestion
                'modal.title' => str_replace('|', ' | ', $this->transChoice('Are you sure you want to delete the selected image?|Are you sure you want to delete the %filesNb% selected images?', 'Admin.Catalog.Notification')),
```

If the separator needs to be adapted you can do something like this.

Reply:
Looks tricky, but it seems viable